python_modules/dagster/dagster/core/asset_defs/asset.py

```diff
@@ -0,0 +1,35 @@
+from typing import Mapping
+
+from dagster.core.definitions import OpDefinition
+from dagster.core.definitions.events import AssetKey
+
+
+class AssetsDefinition:
```

Comment:
nit, what's the naming choice here?  why not `AssetDefinition`?

Reply:
Each one of these can include multiple assets that are computed from the same op and same dependencies